ci/test/00_setup_env_i686_multiprocess.sh

```diff
@@ -14,4 +14,4 @@ export DEP_OPTS="DEBUG=1 MULTIPROCESS=1"
 export GOAL="install"
 export BITCOIN_CONFIG="--enable-debug CC='clang -m32' CXX='clang++ -m32' \
 LDFLAGS='--rtlib=compiler-rt -lgcc_s' CPPFLAGS='-DBOOST_MULTI_INDEX_ENABLE_SAFE_MODE'"
-export TEST_RUNNER_ENV="BITCOIND=bitcoin-node"
```

Comment:
`ci/test/00_setup_env.sh` still contains `export TEST_RUNNER_ENV=${TEST_RUNNER_ENV:-}` which can be dropped as well coz it is the only remaining reference to `TEST_RUNNER_ENV`.